LICENSE

```diff
@@ -0,0 +1,22 @@
+The MIT License (MIT)
+
+Copyright (c) 2011-2020 cdnjs team
```

Comment:
I think it's usually cdnjs contributors

Reply:
I think contributors might imply that anyone who makes a PR holds copyright. I can just remove `team` to make it simpler.

(This is what we have in cdnjs/cdnjs currently)